djangocms_navigation/admin.py

```diff
@@ -19,9 +22,84 @@ def save_model(self, request, obj, form, change):
             )
         super().save_model(request, obj, form, change)
 
+    def get_list_display(self, request):
```

Comment:
If you're not adding any logic to modify the contents of list_display, consider just setting list_display attribute